### ISSUE TITLE
Using SystemColors.MenuHighlight with SystemColors.MenuColor does not meet acessibility contrast ratio

### ISSUE THREAD

**Issue by jordynniara** [dotnet/wpf#6870]
* .NET Core Version: 6.0.7
* Windows version: Windows 10.0.19044
* Does the bug reproduce also in WPF for .NET Framework 4.8?: N/A
* Is this bug related specifically to tooling in Visual Studio (e.g. XAML Designer, Code editing, etc...)? No
* Security issues and bugs should be reported privately, learn more via our [responsible disclosure guidelines](https://github.com/dotnet/wpf/blob/main/README.md#reporting-security-issues-and-security-bugs). N/A

 
 
We have a .Net 6 WPF application that utilizes the SystemColors library for it’s theme coloring. In trying to be S630 A11y compliant, we found that using the SystemColors.MenuHighlight alongside SystemColors.MenuColor does not meet required contrast ratio of 3.1 for graphical objects. The element in question is a window that uses SystemColors.MenuColor as it's background and text box within the window that uses SystemColors.MenuHighlight as its border color when hovered.
 
 **Actual behavior:** Testing SystemColors.MenuHighlight (#3399FF) alongside SystemColors.MenuColor (#F0F0F) yields a Contrast ratio of 2.58.1
 
 **Expected behavior:** SystemColors.MenuHighlight used alongside SystemColors.MenuColor meets a minimum contrast ratio of 3.1 for graphical objects as they are related colors and likely to be used together often. This holds true of all SystemColors in a similar relationship.
 
 **Minimal repro:** Use the any color contrast tool (like Accessibility Insights for Windows - Color Contrast) to test the ratio of the two colors
![image](https://user-images.githubusercontent.com/27777889/182879902-fe3ed2db-2ffc-4cd4-97a3-fa0a568b11a8.png)
